docs/details/pmm_components_and_versions.md

```diff
@@ -0,0 +1,21 @@
+# PMM components and versions
+
+The following table lists all the PMM client/server components and their versions:
+
+
+| Component| Version| URL                                           | GitHub URL                                                                                                  |
+|----------|--------|---------------------------------------------- |-------------------------------------------------------------------------------------------------------------|
+| Grafana  | 9.1    | [Grafana Documentation](https://grafana.com/docs/grafana/latest/)|[Github Grafana](https://github.com/percona-platform/grafana)|                                          
+| VictoriaMetrics| 1.77.1    | [VictoriaMetrics Documentation](https://docs.victoriametrics.com/)|[Github VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics)    |                                          
+| Nginx    | 1.20.1 | [Nginx Documentation](http://nginx.org/en/docs/)|[Github Nginx](https://github.com/nginx/nginx-releases)                                                    |                                          
+| Percona Distribution for PostgreSQL  | 14.1    | [Percona Distribution for PostgreSQL 14 Documentation](https://www.percona.com/doc/postgresql/LATEST/index.html)|              |                                          
+| Clickhouse| 21.3.14.1 |[ClickHouse Documentation Documentation](https://clickhouse.com/docs/en/)|[Github ClickHouse](https://github.com/ClickHouse/ClickHouse)|                                          
+| PerconaToolkit  | 3.4.0    | [Percona Toolkit Documentation](https://www.percona.com/doc/percona-toolkit/3.0/index.html)|[Github Percona Toolkit](https://github.com/percona/percona-toolkit)|                                          
+| Alertmanager  | 0.22.0   | [Alertmanager Documentation](https://prometheus.io/docs/alerting/latest/alertmanager/)|[Github Alertmanager](https://github.com/prometheus/alertmanager)|                                          
```

Comment:
@artemgavrilov please review this version, if it is correct.

Reply:
All good here